MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -159,14 +166,17 @@ void LayeredVolume::removeCongruentElements(std::size_t nLayers, std::size_t nEl
 			{
 				delete _elements[upper_offset+j];
 				_elements[upper_offset+j] = nullptr;
+				_materials[upper_offset+j] = -1;
```

Comment:
if the upper and lower facet of the element collapse, element + mat entry are marked for deletion. added a comment. this has no relevance for dealing with no data values.